src/modules/TeamsList/components/Teams/components/TeamItem/TeamItem.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FC, useState } from 'react';
+import { TeamItemStyled, TeamItemTableWrapper } from './styled';
+import { User } from 'types';
+import { MembersListToggle } from '../MemberListToggle/MemberListToggle';
+import { LIGHT_TEXT_COLOR } from 'appConstants/colors';
+import { TeamUserTable } from '../TeamUserTable/TeamUserTable';
+
+type TeamItemProps = {
+  name: string;
+  description?: string;
+  countMember: number;
+  members: User[];
+};
+
+export const TeamItem: FC<TeamItemProps> = ({
+  name,
+  countMember,
+  description,
+  members,
+}) => {
+  const [isOpen, setToggle] = useState(false);
+  const toggleListHandler = () => {
+    setToggle(!isOpen);
+  };
+  return (
+    <TeamItemStyled>
+      <div className={'teamItem__header'}>
+        <div className={'teamItem__name'}>{name}</div>
+        <div className={'teamItem__description'}>
+          {description ? description : null}
```

Comment:
if we have no description, do we need this div?
maybe `{description ? <div className={'teamItem__description'}>{description}</div>} ` is better?

Reply:
corrected it {desc && <div>...